src/test/rmc/ArithOperators/unsafe_mul_fail.rs

```diff
@@ -6,7 +6,7 @@
 // rmc-flags: --function check_mul
 // compile-flags: --crate-type lib
 
-pub fn check_add(a: u8, b: u8) {
+pub fn check_mul(a: u8, b: u8) {
```

Comment:
Thought we had fixed this 😞

Reply:
Me too! I would like to improve our test framework to make sure this kind of error gets caught.